src/evict.c

```diff
@@ -574,7 +574,7 @@ int performEvictions(void) {
     int prev_core_propagates = server.core_propagates;
     serverAssert(server.also_propagate.numops == 0);
     server.core_propagates = 1;
-    server.propagate_no_multi = 1;
+    server.in_nested_call++;
```

Comment:
why?

Reply:
Without mark that we are on nested call, the first `RM_Call` that will be invoke by the module will cause the data to be replicated. If the module perform more then one `RM_Call`, those will not be replicated in a single `Multi + Exec`.